Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -54,6 +55,8 @@
 #include <map>
 #include <unordered_map>
 #include <vector>
+#include "FT0Base/Geometry.h"
```

Comment:
Hi @mbroz84. It seems that `FT0Base/Geometry.h` is included twice here :)

Reply:
Right, fixed.